openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.component.tsx

```diff
@@ -610,39 +610,46 @@ const DashboardDetails = ({
         dataIndex: 'tags',
         key: 'tags',
         width: 300,
-        render: (text, record, index) => (
-          <div
-            className="relative tableBody-cell"
-            data-testid="tags-wrapper"
-            onClick={() => handleTagContainerClick(record, index)}>
-            {deleted ? (
-              <div className="tw-flex tw-flex-wrap">
-                <TagsViewer sizeCap={-1} tags={text || []} />
-              </div>
-            ) : (
-              <TagsContainer
-                editable={editChartTags?.index === index}
-                isLoading={isTagLoading && editChartTags?.index === index}
-                selectedTags={text as EntityTags[]}
-                showAddTagButton={
-                  dashboardPermissions.EditAll || dashboardPermissions.EditTags
-                }
-                size="small"
-                tagList={tagList}
-                type="label"
-                onCancel={() => {
-                  handleChartTagSelection();
-                }}
-                onSelectionChange={(tags) => {
-                  handleChartTagSelection(tags, {
-                    chart: record,
-                    index,
-                  });
-                }}
-              />
-            )}
-          </div>
-        ),
+        render: (tags: Dashboard['tags'], record, index) => {
+          tags?.sort((tag1, tag2) =>
+            tag1.tagFQN.toLowerCase() < tag2.tagFQN.toLowerCase() ? -1 : 1
+          );
```

Comment:
- write a reuseable comparator function to sort the tags
- Pull out sorting from the render, Sort once and pass it to the table so we won't be ending up sorting again & again